### ISSUE TITLE
Cross entropy function jumps

### ISSUE THREAD

**Issue by masdevas** [masdevas/SingleNeuronTraining#3]
Sometimes loss function have jumps between almost optimal value and very large. At such moments process of optimize doesn't converge.

**Comment by masdevas** [comment]
Resolving by decreasing learning rate for each age.
